helm/vitess/templates/_vttablet.tpl

```diff
@@ -321,10 +414,16 @@ spec:
         -db-config-filtered-uname "vt_filtered"
         -db-config-filtered-dbname "vt_{{$keyspace.name}}"
         -db-config-filtered-charset "utf8"
-        -enable_semi_sync
         -enable_replication_reporter
-{{ if $orc.enabled }}
+{{ if $defaultVttablet.enableSemisync }}
+  {{ if gt $totalTabletCount 1 }}
```

Comment:
I forgot until you brought it up in another context that `rdonly` tablets don't ACK. So we should actually check the replica count of only the `replica` type tablets to avoid getting stuck during ISM.

Reply:
What would you think about eliminating that check altogether? It felt somewhat weird to not enable semisync when the user explicitly enabled it.

Reply:
Yeah that sounds good to me. But maybe add a comment above `enableSemiSync` in `values.yaml` that you need at least 2 replica-type (master-eligible) tablets.

Reply:
Done